src/api/MainSplitter.cc

```diff
@@ -0,0 +1,53 @@
+//
+// Created by prova on 01.04.21.
+//
```

Comment:
Maybe you can use the new license header for these new files? :)

Reply:
Sure